tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -1,4 +1,4 @@
-import React, { useRef } from 'react'
+import React, { useRef, useEffect, useState } from 'react'
```

Comment:
```suggestion
import React, { useRef, useEffect } from 'react'
```
useState appears to be unused